modules/travel_pay/app/services/travel_pay/claims_service.rb

```diff
@@ -14,6 +14,21 @@ def get_claims(current_user)
       }
     end
 
+    def get_claim_by_id(current_user, claim_id)
```

Comment:
🤔 This is making me rethink the whole refactor to include the tokens also as params.... At what point is it too many params to pass to a function call?

Reply:
I should have said earlier, but I don't plan on merging these until you've merged yours, so the burden is on my PRs to fix merge conflicts and figure out that stuff.

That said, I don't like any more than 3 arguments to a method. There are options:
1. dump all the necessary params into an `options` argument. This is sorta low-brow and it hides the level at which the arguments are needed
2. Create the clients at the controller level, constructor inject them into the services. Now there's no need to pass the tokens, and we can focus on passing query/path params. Probably the OO way to do it, but not completely thought out yet

Reply:
we can maybe skip the DI stuff and instantiate the clients in their respective services as instance variables.